### PR TITLE
Update delete-cluster runbook

### PR DESCRIPTION
**Pull request by vijay-veeranki**
Added delete cluster using concourse fly commands

### CODE REVIEW

File: runbooks/source/delete-cluster.html.md.erb [hunk 1]
```diff
@@ -46,6 +46,36 @@ If any steps fail:
...
 
+## Delete the cluster using concourse fly commands
+
+In case you prefer concourse pipeline to destroy the cluster, these are the steps to follow, to delete the cluster using "concourse fly commands"
+
+First, cd to the working copy of the concourse [pipelines repo][pipelines repo]. Make below two changes to the [create-test-destroy.yaml][create-test-destroy] file.
+
+* Add CLUSTER_NAME [var][concourse-vars] under params:
+  ```
+  params:
```
Comment (digitalronin) [inline runbooks/source/delete-cluster.html.md.erb:57]: I'm not clear if you mean to put *literally* this into the pipeline definition, or if you're supposed to replace "CLUSTER_NAME" with the name of the cluster you're deleting. Please could you clarify that?

File: runbooks/source/delete-cluster.html.md.erb [hunk 2]
```diff
@@ -46,6 +46,36 @@ If any steps fail:
...
+In case you prefer concourse pipeline to destroy the cluster, these are the steps to follow, to delete the cluster using "concourse fly commands"
+
+First, cd to the working copy of the concourse [pipelines repo][pipelines repo]. Make below two changes to the [create-test-destroy.yaml][create-test-destroy] file.
+
+* Add CLUSTER_NAME [var][concourse-vars] under params:
+  ```
+  params: 
+      CLUSTER_NAME: ((CLUSTER_NAME)) 
+  ```
+* Comment out "export $(cat keyval/keyval.properties | grep CLUSTER_NAME )" under args: in destroy-cluster job.
```
Comment (digitalronin) [inline runbooks/source/delete-cluster.html.md.erb:60]: Can you make this a link to the relevant lines of the file on github?

File: runbooks/source/delete-cluster.html.md.erb [hunk 3]
```diff
@@ -46,6 +46,36 @@ If any steps fail:
...
+
+
+Run the below commands updating the <cluster-name-to-be-deleted>. The first fly command will apply the changes made for the [create-test-destroy.yaml][create-test-destroy] file and set the CLUSTER_NAME [static var][concourse-static-vars], the second command will trigger the destroy-cluster job using the CLUSTER_NAME provided.
+
+```
+fly -t manager sp -p create-test-destroy -c create-test-destroy.yaml -v CLUSTER_NAME=<cluster-name-to-be-deleted>
+fly -t manager trigger-job -j create-test-destroy/destroy-cluster
+
+```
+Note: After the destroy-cluster job completed sucessfully, run the [bootstrap pipleine][bootstrap pipleine] to discard the changes made to [create-test-destroy.yaml][create-test-destroy] file.
```
Comment (digitalronin) [inline runbooks/source/delete-cluster.html.md.erb:77]: Could you add the fly command syntax required to do this? That way, people following this runbook can do everything via the command-line.

**Comment by vijay-veeranki** [comment]
Thanks @digitalronin for the review comments, updated considering your suggestions. Can you please review it again.

**Pull request merged**